services/app-web/src/s3/s3Amplify.ts

```diff
@@ -62,22 +64,38 @@ function newAmplifyS3Client(bucketName: string): S3ClientT {
             }
         },
 
-        deleteFile: async (filename: string): Promise<void | S3Error> => {
-            try {
-                await Storage.remove(filename)
-                return
-            } catch (err) {
-                assertIsS3PutError(err)
-                if (err.name === 'Error' && err.message === 'Network Error') {
-                    console.log('Error deleting file', err)
-                    return {
-                        code: 'NETWORK_ERROR',
-                        message: 'Error deleting file from the cloud.',
+        deleteFile: async (
```

Comment:
I don't think this checking logic belongs here. One reason this sticks out to me is that you needed to duplicate the logic in our Amplify and our Local deleteFile implementations. Another reason is that this s3 package is a wrapper around a dependency, we want this code to expose the functionality of the dependency and not much more. Logic like when to delete a file belongs in our apps "business logic" layer rather than a dependency wrapper.